pkg/apis/navigator/types.go

```diff
@@ -206,6 +208,12 @@ type ElasticsearchPilotStatus struct {
 	Version   semver.Version
 }
 
+type CassandraPilotStatus struct {
+	// The version reported by the Cassandra process.
+	// nil indicates that the version is not yet known / not yet reported.
+	Version *semver.Version
```

Comment:
This comment is best on the v1alpha1 type (especially the extra info about nil). We won't be generating docs from internal types, only the external versions.